core/templates/pages/exploration-player-page/layout-directives/feedback-popup.component.html

```diff
@@ -47,8 +47,9 @@
     position: absolute;
   }
   input {
+    margin-right: 5px;
```

Comment:
@aks681 Yes, it looks fine with margin-right only. If margin-left is also given then the checkbox will move out and it will not be properly left-aligned